ReactWindows/ReactNative.Shared/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,229 @@
+                case ImportantForAccessibility.Auto:
+                    uiElement.ClearValue(AutomationProperties.AccessibilityViewProperty);
+                    SetChildrenAccessibilityViewFromImportantForAccessibility(uiElementAutomationPeer);
+                    break;
+                case ImportantForAccessibility.No:
+                    AutomationProperties.SetAccessibilityView(uiElement, AccessibilityView.Raw);
+                    SetChildrenAccessibilityViewFromImportantForAccessibility(uiElementAutomationPeer);
+                    break;
+                case ImportantForAccessibility.NoHideDescendants:
+                    AutomationProperties.SetAccessibilityView(uiElement, AccessibilityView.Raw);
+                    SetChildrenAccessibilityView(uiElementAutomationPeer, AccessibilityView.Raw);
+                    break;
+                case ImportantForAccessibility.Yes:
+                    AutomationProperties.SetAccessibilityView(uiElement, AccessibilityView.Content);
+                    SetChildrenAccessibilityView(uiElementAutomationPeer, AccessibilityView.Raw);
+                    break;
+                default:
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Attached property used to store ImportantForAccessibility value in native controls.
+        /// </summary>
+        public static readonly DependencyProperty ImportantForAccessibilityAttachedProperty = DependencyProperty.RegisterAttached(
+                                                                                                "ImportantForAccessibilityAttached",
+                                                                                                typeof(ImportantForAccessibility),
+                                                                                                typeof(UIElement),
+                                                                                                new PropertyMetadata(ImportantForAccessibility.Auto));
+
+        /// <summary>
+        /// ImportantForAccessibilityAttached property setter.
+        /// </summary>
+        /// <param name="element"></param>
+        /// <param name="value"></param>
+        public static void SetImportantForAccessibilityAttached(UIElement element, ImportantForAccessibility value)
+        {
+            element.SetValue(ImportantForAccessibilityAttachedProperty, value);
+        }
+
+        /// <summary>
+        /// ImportantForAccessibilityAttached property getter.
+        /// </summary>
+        /// <param name="element"></param>
+        /// <returns></returns>
+        public static ImportantForAccessibility GetImportantForAccessibilityAttached(UIElement element)
+        {
+            return (ImportantForAccessibility)element.GetValue(ImportantForAccessibilityAttachedProperty);
+        }
+
+        /// <summary>
+        /// Recursively sets AccessibilityView to accessibilityView for parentAutomationPeer children.
```

Comment:
>parentAutomationPeer [](start = 72, length = 20)

you can use <paramref> to avoid the xmldoc becoming stale if parameter names change.

Reply:
Fixed.

---
In reply to: [163125280](https://github.com/Microsoft/react-native-windows/pull/1595#discussion_r163125280) [](ancestors = 163125280)